src/System.Reflection.Metadata/src/System/Reflection/PortableExecutable/PEReader.cs

```diff
@@ -21,6 +21,11 @@ namespace System.Reflection.PortableExecutable
     /// </remarks>
     public sealed class PEReader : IDisposable
     {
+        /// <summary>
+        /// True if the PE image has been loaded into memory by the OS loader.
+        /// </summary>
+        public bool IsLoadedImage { get; }
```

Comment:
In `PEHeaderBuilder`, the constructor argument is named `isImageLoaded`. Should the property and the constructor argument in this class be renamed to be consistent?